<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2012 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>assessFinanceCharges</class>
 <widget class="QWidget" name="assessFinanceCharges">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assess Finance Charges</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>531</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Filter Invoices based on the below criteria.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Customer ID Range</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="_customerIdFrom">
            <property name="text">
             <string>From</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="_customerIdTo">
            <property name="text">
             <string>To</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Statement Cycle Range</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="_assessCycleFrom">
            <property name="text">
             <string>From</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="_assessCycleTo">
            <property name="text">
             <string>To</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="_filterCharges">
          <property name="text">
           <string>Filter Charges</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Click the assess column for each customer for whom you wish to create a finance charge invoice.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="_invoiceList"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Assessment Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="_assessmentDate"/>
        </item>
        <item>
         <widget class="QPushButton" name="_assessCharges">
          <property name="text">
           <string>Assess Charges</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="_unmarkAllInvoices">
            <property name="text">
             <string>Unmark All Invoices</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_markAllInvoices">
            <property name="text">
             <string>Mark All Invoices</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
